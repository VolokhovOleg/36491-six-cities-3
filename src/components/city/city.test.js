import City from './city';

const onCityClick = () => {};
const name = `Moscow`;

it(`<City /> snapshot`, () => {
  const tree = renderer
    .create(<City
      onCityClick={onCityClick}
      name={name}
      state={false}
    />
    ).toJSON();

  expect(tree).toMatchSnapshot();
});
