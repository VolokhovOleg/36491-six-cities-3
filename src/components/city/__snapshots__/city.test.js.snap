// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<City /> snapshot 1`] = `
<li
  className="locations__item"
  onClick={[Function]}
>
  <a
    className="locations__item-link tabs__item  "
  >
    <span>
      Moscow
    </span>
  </a>
</li>
`;
